Task_4/scripts/generateTask.js

```diff
@@ -0,0 +1,103 @@
+const cols = document.querySelectorAll(".col");
+
+function findValue(elem) {
+  return elem.innerHTML;
+}
+
+function isPrivate(value) {
+  return value ? "Private" : "Public";
+}
+
+function makeCard(obj, parent) {
```

Comment:
better use innerHTML this static HTMLcode. For example = elem.innerHTML = 'div...../div'